source/update.rst

```diff
@@ -29,7 +29,12 @@ Here are the steps to update GLPI:
 
 .. note::
 
-    Update process will automatically disable your plugins.
+    Before GLPI 11.0, all the plugins were disabled during the ppdate process.
+    
+    Since GLPI 11.0, as soon as a new version of GLPI files is detected, the plugins execution is suspended.
+    Their execution will be resumed after the update only in case of a minor/bugfixes update.
+    Otherwise you will have to do it manually using the ``php bin/console plugin:resume_execution`` :ref:`command line tool <_cdline_suspend_resume_plugins>`
```

Comment:
```suggestion
    Otherwise, you will have to do it manually using the ``php bin/console plugin:resume_execution`` :ref:`command line tool <_cdline_suspend_resume_plugins>`
```